backend/person/admin.py

```diff
@@ -42,6 +47,14 @@ class AgentDescriptionLocationAdmin(admin.StackedInline):
     fields = ["location"] + core_admin.description_field_fields
     extra = 0
 
+    def get_field_queryset(
+        self, db, db_field: RelatedField, request: HttpRequest
+    ) -> QuerySet:
+        if db_field.name == "location":
```

Comment:
The type hints here hide the fact that request is typed as `HttpRequest | None`. I have no idea when this would be `None`, but best to handle this case anyway.